doc/ghdl.1

```diff
@@ -94,10 +94,8 @@ Dump VCD to FILENAME (a waveform dump, viewable with\-\-for instance\-\-\fBgtkwa
 Back annotate SDF onto design using TYPE (min|typ|max), instance PATH, and SDF file FILENAME, i.e. \fI./sdf_design \-\-sdf=typ=top/inst=inst.sdf\fP
 
 .SH "SEE ALSO"
-.TP
-.B gtkwave (1)
+.BR gtkwave (1)
```

Comment:
This file is actually auto-generated through Sphinx. It is included in the repo, so that packagers don't need to execute Sphinx explicitly, allowing them to package GHDL without using Python. However, the changes done here might be reverted at any time.

@andreasbombe can you afford to execute Sphinx for updating `ghdl.1` during packaging?

Reply:
I didn't realize that. Sphinx would be an additional build dependency, but then again ghdl is already heavy. Anyway, it was just a quick fix I had in addition to some changes for documenting Debian stuff.

Reply:
I didn't realize that. Sphinx would be an additional build dependency, but then again ghdl is already heavy. Anyway, it was just a quick fix I had in addition to some changes for documenting Debian stuff. I'll look into how to deal with this going forward.